src/components/Facility/FacilityCard.tsx

```diff
@@ -222,38 +223,49 @@ export const FacilityCard = (props: {
                   </div>
                   <div className="flex flex-wrap gap-2">
                     {["DistrictAdmin", "StateAdmin"].includes(userType) && (
-                      <ButtonV2
+                      <Button
                         id="facility-notify"
-                        ghost
-                        border
-                        className="h-[38px]"
+                        variant={"outline_primary"}
+                        className="h-[38px] tooltip"
                         onClick={(_) => setNotifyModalFor(facility.id)}
+                        aria-label={t("notify")}
+                        role="button"
                       >
                         <CareIcon icon="l-megaphone" className="text-lg" />
-                        <span className="hidden md:block">Notify</span>
-                      </ButtonV2>
+                        <span className="hidden sm:inline">{t("notify")}</span>
+                        <span className="tooltip-text tooltip-top sm:hidden">
```

Comment:
I'd recommend using shadcn's tooltip component as we are moving towards migrating to those in newer codebases.

Reply:
@rithviknishad UI after applying shadcn component


https://github.com/user-attachments/assets/4123e4be-1bc6-45f1-b933-fb5c415f5bcb